Dockerfile.prod

```diff
@@ -0,0 +1,43 @@
+FROM ubuntu:18.04
+
+MAINTAINER JacobEberhardt <jacob.eberhardt@tu-berlin.de>, Dennis Kuhnert <mail@kyroy.com>
```

Comment:
Could add you there @Schaeff

Reply:
Oops this file was not meant to be commited, I'll remove it